official/resnet/ctl/ctl_benchmark.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
I know this follows a pattern I helped push.  Can we remove this file and move the base class (if needed) into the main benchmark.  I created a [base class](https://github.com/tensorflow/models/blob/master/official/utils/testing/perfzero_benchmark.py) you can extend that does most of what this does.  [Example usage](https://github.com/tensorflow/models/blob/master/official/transformer/v2/transformer_benchmark.py)

Reply:
Done. We might need to update keras to do the same. I can do that in a follow up PR.